src/main/webapp/app/exercises/shared/course-exercises/course-exercise.service.ts

```diff
@@ -158,14 +157,18 @@ export class CourseExerciseService {
 
     convertExerciseDatesFromServer<T extends Exercise>(res: T): T {
         res.releaseDate = convertDateFromServer(res.releaseDate);
+        res.startDate = convertDateFromServer(res.startDate);
         res.dueDate = convertDateFromServer(res.dueDate);
+        res.assessmentDueDate = convertDateFromServer(res.assessmentDueDate);
+        res.exampleSolutionPublicationDate = convertDateFromServer(res.exampleSolutionPublicationDate);
```

Comment:
Unrelated to this PR, but I think the date of the additional test run for PE should also be converted here.

Reply:
This method handles all exercise types, so we cannot do this here